trade-finance-manager-api/src/v1/emails/AIN-MIN-confirmation/gef-facilities-list.js

```diff
@@ -16,71 +16,58 @@ const mapIssuedValue = (hasBeenIssued) => {
   return CONSTANTS.FACILITIES.FACILITY_STAGE_PORTAL.UNISSUED;
 };
 
+/**
+ * @param {unknown} value
```

Comment:
Should this be (looks like) `{String | Boolean}`?

Reply:
`isUsingFacilityEndDate` should be one of `boolean | undefined | null` but since this function can handle any input, `unknown` also captures it.